src/routing/http-routing-v1.md

```diff
@@ -102,6 +102,25 @@ Response limit: 100 providers
 
 Each object in the `Providers` list is a *read provider record*.
 
+## Peers API
+
+### `GET /routing/v1/peers/{peer-id}`
+
+#### Path Parameters
+
+- `peer-id` is the [Peer ID](https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md) to fetch peer records for,
+represented as a CIDv1 encoded with `libp2p-key` codec.
+
+#### Response Status Codes
+
+- `200` (OK): the response body contains the peer record.
+- `404` (Not Found): must be returned if no matching records are found.
+- `422` (Unprocessable Entity): request does not conform to schema or semantic constraints.
+
+#### Response Body
+
+A [`peer` schema record](#peer).
```

Comment:
**Open Question**: should the response body for `/routing/v1/peers` be a single entity? What we want to obtain is the addresses of the peer. But addresses can end up being discovered on the fly. Supporting streaming is reasonably easy and could be part of the spec.

Reply:
cid.contact offers both: `/providers` returns list of all providers and `/providers/<peer-id>` returns a specific one.

Reply:
I updated the Peers API to indicate it can return more than one. It will conform with the streaming API we already have too.